Services/Administration/OJS.Services.Administration.Data/Implementations/SubmissionsDataService.cs

```diff
@@ -105,7 +105,6 @@ protected override Expression<Func<Submission, bool>> GetUserFilter(UserInfoMode
 
         private static IQueryable<Submission> IncludeProblemTestsAndSubmissionTypes(IQueryable<Submission> queryable)
             => queryable
-                .AsNoTracking()
```

Comment:
![image](https://github.com/user-attachments/assets/44306c74-4dc4-4687-94d5-82ce6f5eac60)

We do update operations after this query so we need the submission to be tracked

Reply:
The submission is retrieved using OneById in SubmissionsPermissionsService where we begin tracking the entity. Then when we retrieve the submission here again and do not include tracking, but try to update the submission in further operations, the following error occurs:

`
The instance of entity type 'Submission' cannot be tracked because another instance with the same key value for {'Id'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values.
`

![image](https://github.com/user-attachments/assets/f7de034f-2651-4900-a8a3-8690acdce5bc)